skate-plugin/src/main/resources/messages/skateBundle.properties

```
@@ -12,3 +12,10 @@ skate.configuration.featureFlagFilePattern.title=Provide a regex pattern to iden
 skate.configuration.featureFlagBaseUrl.title=Base URL for feature flag linking. The flag key will be appended to the end of this
 skate.configuration.featureFlagAnnotation.title=Feature flag annotation's fully-qualified name (com.example.FeatureFlag)
 skate.configuration.featureFlagFieldEmpty.error=Field cannot be empty when feature flag linking is enabled
+skate.configuration.enableTracing.title=Enable Data Tracking
+skate.configuration.enableTracing.description=If enabled, allow sending plugin usage statistics to Slack
```

Let's maybe specify it's to devxp

```suggestion
skate.configuration.enableTracing.description=If enabled, allow sending plugin usage statistics to Slack Mobile DevXP
```